rasa/importers/rasa.py

```diff
@@ -32,9 +32,7 @@ def __init__(
 
         self._domain_path = domain_path
 
-        self._story_files, self._nlu_files = data.get_core_nlu_files(
-            training_data_paths
-        )
+        self.story_files, self.nlu_files = data.get_core_nlu_files(training_data_paths)
```

Comment:
Changed these to public because i'm using them somewhere else (and can see it being helpful in other places in the future). Going into a minor, do i need to make any sort of deprecation warning? It's in the changelog as well.

Reply:
Why are you making this public? Why not using `get_stories` and `get_nlu_data`?

Reply:
Get_stories goes through the whole thing and creates the story graph, which is the part that takes forever right? Seems unnecessary just to know if there is core data or not, when the file importer already has this information (and is just hiding it) -- it more closely reflects the information I was already using

Reply:
Hm okay not sure i'd agree because if you don't pass data, it just auto-picks the --data path which could be confusing but that's a separate thing. If I used it and passed a model, i (clearly) expect it to match the model I gave, but can see how that isn't technically feasible.

maybe we should throw an INFO or something about where the visualization data is coming from if the model is passed explicitly

Reply:
nevermind, i was misinterpreting the method and thought creating the story graph included generating all of the trackers, but I was wrong. I'll change that back